src/components/MySpace/MySpace.tsx

```diff
@@ -101,108 +114,119 @@ const MySpace = ({ bookmarks }: { bookmarks: BookmarkRecords }) => {
           {data &&
             data.mySpace &&
             data.mySpace.map((widget: Widget) => (
+              <React.Fragment key={`widget_${widget._id}`}>
+                {widget.type === 'GuardianIdeal' ? (
+                  <Grid className={styles.guardianIdeal}>
+                    <GuardianIdealCarousel
+                      ideals={GuardianIdealPillars}
+                      widget={widget}
+                    />
+                  </Grid>
+                ) : (
+                  <Grid tabletLg={{ col: 6 }} desktopLg={{ col: 4 }}>
+                    {widget.type === 'News' && <NewsWidget widget={widget} />}
```

Comment:
Question: I was wondering if there was a reason we couldn't handle the guardian ideals similar to the news widget. Is it because of the size of the grid needed?

Reply:
Yeah the `Grid` just needs more space. I initially tried to treat it the same as the news widget, but it wants to render it in the same space as the collections, which just doesn't work. I think as we add more widgets, and especially as we look at implementing drag-and-drop for collections, we may need to rethink this structure.